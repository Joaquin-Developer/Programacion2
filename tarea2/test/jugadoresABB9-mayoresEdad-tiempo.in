crearArbolJugadores
mayoresEdadArbolJugadoresTiempo 10000 2 19900
liberarArbolJugadores

Fin
