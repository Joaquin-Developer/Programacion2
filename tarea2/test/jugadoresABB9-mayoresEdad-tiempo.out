1>2>Jugador 9951: Juan, Uruguay, 19902 años
Jugador 9952: Juan, Uruguay, 19904 años
Jugador 9953: Juan, Uruguay, 19906 años
Jugador 9954: Juan, Uruguay, 19908 años
Jugador 9955: Juan, Uruguay, 19910 años
Jugador 9956: Juan, Uruguay, 19912 años
Jugador 9957: Juan, Uruguay, 19914 años
Jugador 9958: Juan, Uruguay, 19916 años
Jugador 9959: Juan, Uruguay, 19918 años
Jugador 9960: Juan, Uruguay, 19920 años
Jugador 9961: Juan, Uruguay, 19922 años
Jugador 9962: Juan, Uruguay, 19924 años
Jugador 9963: Juan, Uruguay, 19926 años
Jugador 9964: Juan, Uruguay, 19928 años
Jugador 9965: Juan, Uruguay, 19930 años
Jugador 9966: Juan, Uruguay, 19932 años
Jugador 9967: Juan, Uruguay, 19934 años
Jugador 9968: Juan, Uruguay, 19936 años
Jugador 9969: Juan, Uruguay, 19938 años
Jugador 9970: Juan, Uruguay, 19940 años
Jugador 9971: Juan, Uruguay, 19942 años
Jugador 9972: Juan, Uruguay, 19944 años
Jugador 9973: Juan, Uruguay, 19946 años
Jugador 9974: Juan, Uruguay, 19948 años
Jugador 9975: Juan, Uruguay, 19950 años
Jugador 9976: Juan, Uruguay, 19952 años
Jugador 9977: Juan, Uruguay, 19954 años
Jugador 9978: Juan, Uruguay, 19956 años
Jugador 9979: Juan, Uruguay, 19958 años
Jugador 9980: Juan, Uruguay, 19960 años
Jugador 9981: Juan, Uruguay, 19962 años
Jugador 9982: Juan, Uruguay, 19964 años
Jugador 9983: Juan, Uruguay, 19966 años
Jugador 9984: Juan, Uruguay, 19968 años
Jugador 9985: Juan, Uruguay, 19970 años
Jugador 9986: Juan, Uruguay, 19972 años
Jugador 9987: Juan, Uruguay, 19974 años
Jugador 9988: Juan, Uruguay, 19976 años
Jugador 9989: Juan, Uruguay, 19978 años
Jugador 9990: Juan, Uruguay, 19980 años
Jugador 9991: Juan, Uruguay, 19982 años
Jugador 9992: Juan, Uruguay, 19984 años
Jugador 9993: Juan, Uruguay, 19986 años
Jugador 9994: Juan, Uruguay, 19988 años
Jugador 9995: Juan, Uruguay, 19990 años
Jugador 9996: Juan, Uruguay, 19992 años
Jugador 9997: Juan, Uruguay, 19994 años
Jugador 9998: Juan, Uruguay, 19996 años
Jugador 9999: Juan, Uruguay, 19998 años
Bien.
3>4>Fin.
